ga4gh/datamodel/variants.py

```diff
@@ -415,6 +427,28 @@ def convertVariant(self, record, callSetIds):
             sampleIterator += 1  # REMOVAL
         return variant
 
+    def getVariant(self, compoundId):
+        if compoundId.chrom in self._chromFileMap:
+            varFileName = self._chromFileMap[compoundId.chrom]
+        else:
+            raise exceptions.ObjectNotFoundException(compoundId)
+        pos = int(compoundId.pos)
+        referenceName, startPosition, endPosition = \
+            self.sanitizeVariantFileFetch(
+                compoundId.chrom, pos, None)
+        # TODO vcf.fetch's startPosition arg seems to be exclusive,
```

Comment:
I don't know --- perhaps this is because VCFs have a 1-based coordinate system rather than 0 based? We should have hit this already if it wasn't being managed.